src/test/java/org/fundacionjala/coding/christian/Kata6Test.java

```diff
@@ -0,0 +1,64 @@
+package org.fundacionjala.coding.christian;
+
+import org.junit.Before;
+import org.junit.Test;
+import java.util.Arrays;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Powered by Christian Galarza.
+ */
+public class Kata6Test {
+    private Kata6 sort;
+
+    /**
+     *
+     */
+    @Before
+    public void setup() {
+        sort = new Kata6();
+    }
+
+
+    private final Integer[] KEEPER_1 = new Integer[]{1, 2, 7, 4, 5, 6, 3, 8, 9};
+    private final Integer[] EARTH_1 = new Integer[]{1, 2, 3, 4, 5, 6, 7, 8, 9};
+    private final Integer[] KEEPER_2 = new Integer[]{12, 14, 13};
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this final field static too. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1170)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this field "KEEPER_2" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00116)